spec/features/student_user_registers_for_courses_spec.rb

```diff
@@ -11,7 +11,7 @@
   [x] Registering for courses creates the appropriate course_registrations objects
 } do
 
-  scenario 'user with an associated student registers for a class' do
+  pending 'user with an associated student registers for a class' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.